semantic_conventions/trace/rpc.yaml

```
@@ -33,3 +33,14 @@ groups:
           - net.peer.ip
           - net.peer.name
         - include: network
+    - id: rpc.grpc
+      prefix: rpc.grpc
+      extends: rpc
+      brief: 'Call-level attributes for gRPC'
+      attributes:
+        - id: status_code
+          type: number
+          required: always
+          tag: call-level-tech-specific
+          brief: "the status code number of the gRPC request."
+          examples: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16]
```

OK, unfortunately the examples are kinda pointless here, but since they are required, maybe use a bit fewer at least.

```suggestion
          examples: [0, 1, 16]
```

---

not strongly opinionated, can do. I figured since it was short might as well enumerate all values.

---

No strong opinion either. Fine for me to keep as-is.

---

for posterity, I did go with the suggestion because otherwise the generated table results in a very tall row (due to each example being on a different row).